music_album/music_album.rb

```diff
@@ -1 +1,15 @@
+require 'json'
+require_relative '../item'
 
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date, id = nil, archived: false)
```

Comment:
Why don't we make the ID random to make everything tally, that is, not setting any value for it, whether nil or anything. The instance will just be set to @id = Random.rand(1..1000)